netconf/message/commit.go

```diff
@@ -62,3 +64,49 @@ func NewCommit(confirmed bool, confirmTimeout int) *Commit {
 	rpc.MessageID = uuid()
 	return &rpc
 }
+
+// NewConfirmedCommit creates a confirmed-commit message with an optional
+// confirm-timeout and persist token.
+//
+// Example confirmed-commit with persist:
+// <rpc message-id="101">
+// ..<commit>
+// ....<confirmed/>
+// ....<confirm-timeout>120</confirm-timeout>
+// ....<persist>IQ,d4668</persist>
+// ..</commit>
+// </rpc>
+func NewConfirmedCommit(confirmTimeout int, persist string) *Commit {
```

Comment:
if `confirm-commit` is optional, you can pass a pointer to it like that:

```suggestion
func NewConfirmedCommit(confirmTimeout *int, persist string) *Commit {
```

Then you only check if it's `nil` or not and directly assign it to the field (see suggestion below)